src/io/flutter/jxbrowser/EmbeddedBrowser.java

```diff
@@ -20,19 +21,30 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import static com.teamdev.jxbrowser.engine.RenderingMode.HARDWARE_ACCELERATED;
+import static com.teamdev.jxbrowser.engine.RenderingMode.OFF_SCREEN;
 
 public class EmbeddedBrowser {
   public void openPanel(ContentManager contentManager, String tabName, String url) {
-    final EngineOptions options =
+    EngineOptions options =
       EngineOptions.newBuilder(HARDWARE_ACCELERATED).build();
-    final Engine engine = Engine.newInstance(options);
-    final Browser browser = engine.newBrowser();
-    browser.settings().enableTransparentBackground();
+    Engine engine = Engine.newInstance(options);
+    Browser browser = engine.newBrowser();
+
+    try {
+      browser.settings().enableTransparentBackground();
+    } catch (UnsupportedRenderingModeException ex) {
+      options =
```

Comment:
does HARDWARE_ACCELERATED without the transparent background work on windows? Perhaps add a comment as the placement of the try-catch makes it look like that is the problem not the HARDWARE_ACCELERATED option. I'll put it this way, if I had to pick, HARDWARE_ACCELERATED over enableTransparentBackground as one impacts performance and the other just causes a UI flicker.

Reply:
I can skip enabling transparent background, but `HARDWARE_ACCELERATED` mode also causes the window to appear to the left of the panel. Looking into if I can fix this.. maybe a swing issue?

<img width="1280" alt="Annotation 2020-09-09 135226" src="https://user-images.githubusercontent.com/6379305/92653879-d546cf00-f2a3-11ea-8b84-9f8dedd12f8b.png">

Reply:
That seems like it is a jxbrowser bug on windows. Swing shouldn't have many windows specific issues.

Reply:
Hmm.. I'll post an issue and see if they have any advice.